### DISCUSSION TITLE
qmk Caps Word

### DISCUSSION THREAD

**Discussion by l3ztum** [manna-harbour/miryoku#199] [Q&A] [answered]
#51 this feature does not seem work, since all defines in miryoku_layer_alternatives.h use the lefthand-side modifiers from qmk.

https://github.com/manna-harbour/miryoku_qmk/blob/0e24df7769af76681a0c044e281fd6f397706aa0/users/manna-harbour_miryoku/miryoku_babel/miryoku_layer_alternatives.h#L91

**Comment by manna-harbour** [comment]
[Issues](https://github.com/manna-harbour/miryoku/issues) are used to track future work and are not used for support requests. For support, see [Miryoku Discussions and Support](https://github.com/manna-harbour/miryoku#discussions-and-support).

**Comment by manna-harbour** [comment, marked as answer]
See https://github.com/manna-harbour/miryoku_qmk/tree/miryoku/users/manna-harbour_miryoku#caps-word. Caps Word / Caps Lock is on [Nav](https://github.com/manna-harbour/miryoku/tree/master/docs/reference#nav). You don't need to use both Shifts to activate.
